controllers/base-controller.go

```diff
@@ -94,11 +95,12 @@ func (this *baseController) sendFailedResponse(statusCode int, errCode string, r
 	this.sendResponse(d, statusCode)
 }
 
-func (this *baseController) newApiToken(permission string, pl interface{}) (string, error) {
+func (this *baseController) newApiToken(permission string, addr string, pl interface{}) (string, error) {
 	ac := &accessController{
 		Expiry:     util.Expiry(config.AppConfig.APITokenExpiry),
 		Permission: permission,
 		Payload:    pl,
+		RemoteAddr: addr,
```

Comment:
It is better to invoke getRemoteAddr in baseController.newApiToken which matches the function of 'baseController.checkApiReqToken' that also invokes 'getRemoteAddr' itself.